index.html

```diff
@@ -48,4 +48,178 @@ <h2 class="hex connect">LET’S CONNECT</h2>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-3.png" alt="Project-1" />
+            <div class="details">
+            <div class="titles">
+            <h3 class="project-title">Tonic</h3>
+            <ul class="canopy">
+            <li>CANOPY</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-4.png" alt="Project-1" />
+            <div class="details">
+            <div class="titles">
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <ul class="canopy">
+            <li>CANOPY</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            </div>
+            <div class="about-myself">
+            <h2 class="about">About Myself</h2>
+            <p class="text">
```

Comment:
- [ ] Please make the top-right corner of the `about section` round just like in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=uQuKu3bekkovCjcN-0) to make your design look as close to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=uQuKu3bekkovCjcN-0) as possible..
- [ ] Also please make the background color of the `work` section  `#E5E5E5 ` to match the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=uQuKu3bekkovCjcN-0).

Figma design:

<img width="331" alt="Screenshot 2023-03-25 at 5 00 12 AM" src="https://user-images.githubusercontent.com/44798044/227667817-df041a9e-26de-45bb-bdc7-8ef9d1f415db.png">

Your Design:

<img width="355" alt="Screenshot 2023-03-25 at 5 36 15 AM" src="https://user-images.githubusercontent.com/44798044/227668320-ee0902e1-bfef-4cd8-8fd4-f2e72ffc9a50.png">